src/by/epam/ayem/module5/service/DragonsTreasureApp.java

```diff
@@ -42,21 +45,13 @@ public void run() {
 
             switch (choice) {
```

Comment:
What is choise? does it even compile?

Reply:
Yes, why not? Choice is the name of the int variable which I'm getting from the scanner input. It means the choice of a number of a client in the menu.